feg/gateway/services/s6a_proxy/servicers/ul.go

```diff
@@ -213,3 +207,19 @@ func getFeatureListID2(supportedFeatures []SupportedFeatures) *protos.FeatureLis
 	}
 	return protoFeatureList
 }
+
+func getTotalAmbr(ambr AMBR) *protos.UpdateLocationAnswer_AggregatedMaximumBitrate{
```

Comment:
maybe attaching this function to the object could be nice? (totally up to you though0
`func (ambr *AMBR) getProtoAmbr() *protos.UpdateLocationAnswer_AggregatedMaximumBitrate {`

Reply:
sure good idea